File: OOAD/solid-principle-app/src/lsp/refactor/Rectangle.java
package lsp.refactor;

public class Rectangle implements Polygon{
	private int height;
	private int width;
	
	public Rectangle(int height,int width){
		this.height=height;
		this.width=width;
	}
	
	public int getWidth() {
		return width;
	}
	public void setWeight(int width) {
		this.width = width;
	}
	public int getHeight() {
		return height;
	}
	public void setHeight(int height) {
		this.height = height;
	}

	@Override
	public double calculateArea() {
		int area=this.height*this.width;
		return area;
	}
}

File: C#/OOP/SOLID Principle/LiskovSubstitution/LiskovSubstitution/Rectangle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace LiskovSubstitution
{
    class Rectangle : Polygon
    {
        private int height;
        private int width;

        public Rectangle(int height, int width)
        {
            this.height = height;
            this.width = width;
        }

        public int Height { get { return height; } set { height = value; } }

        public int Width { get { return width; } set { width = value; } }

        public double CalculateArea()
        {
            int area = this.height * this.width;
            return area;
        }
    }
}
